jerry-main/main-unix.c

```diff
@@ -418,6 +418,20 @@ check_usage (bool condition, /**< the condition that must hold */
   }
 } /* check_usage */
 
+#ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
+
+/**
+ * The alloc function passed to jerry_create_instance
+ */
+static void *
+instance_alloc (size_t size,
+                void *cb_data_p __attribute__((unused)))
+{
+  return malloc (size);
```

Comment:
Malloc is not necessary available in main-unix.c. Is there a check that system libc must be enabled?